deployments/testnet/tokenBridgeVAAs.csv

```diff
@@ -10,7 +10,7 @@ Algorand (8) Testnet Token Bridge,010000000001006e90fe2a505f0d67b6ca6221a42f5595
 Aurora (9) (NEAR) Testnet Token Bridge,0100000000010096e79a229cdd3717f0b47bf7fb65b88202c4ec5fd9cc7d5f319cbe27b0f57d0c2292204c9df25e060e7e9632a54fefca88f1ca95c76ca083a07a68d59de9964701000000010000000100010000000000000000000000000000000000000000000000000000000000000004000000000227003a00000000000000000000000000000000000000000000546f6b656e4272696467650100000009000000000000000000000000d05ed3ad637b890d68a854d607eeaf11af456fba
 Fantom (10) Testnet Token Bridge,010000000001007bc5bdc2253a6deec065a6e573f7bdd363383b91210054d8ce7558d9b8185a60157370f56f35d03fc434ecf90fc652cf31b9b4a2a15eb43c5ad8a664356ef81e00000000010000000100010000000000000000000000000000000000000000000000000000000000000004000000000351c06c00000000000000000000000000000000000000000000546f6b656e427269646765010000000a000000000000000000000000599cea2204b4faecd584ab1f2b6aca137a0afbe8
 Karura (11) Testnet Token Bridge,01000000000100c01c50054de5f481bb5cd8b4e5d906d86c7307ba969a0f9f042e410b859528387040360954dc4fd4834d4f68dfea9067f3da3bb4af127f61e3ae04f7d216fc300100000001000000010001000000000000000000000000000000000000000000000000000000000000000400000000026bcc5b00000000000000000000000000000000000000000000546f6b656e427269646765010000000b000000000000000000000000d11de1f930ea1f7dd0290fe3a2e35b9c91aefb37
-Acala (12) Testnet Token Bridge,01000000000100595f3b02b450ea71724c09a4b1a071f67eafb1471e0bd0d03d506e11ceedc81d788dd57dc81927cb74970ad6389e369d0589682d176eff033fe140dae1035155000000000100000001000100000000000000000000000000000000000000000000000000000000000000040000000002a49e7a00000000000000000000000000000000000000000000546f6b656e427269646765010000000c000000000000000000000000eba00cbe08992edd08ed7793e07ad6063c807004
+Acala (12) Testnet Token Bridge,01000000000100f14dcd0abe531c59fd5360750781a38c03fc1ba19b4fb39a4e91b2ee32dcaf5f6fad2dc1c83a7d20f7aa9fc8c57ad156aa29e1344bfc4f97123d30ebdea869eb010000000100000001000100000000000000000000000000000000000000000000000000000000000000040000000005c4aa5000000000000000000000000000000000000000000000546f6b656e427269646765010000000c00000000000000000000000064fb09e405d2043ed7785a29e296c766d56f2056
```

Comment:
Wait, this looks like it's for the core address, not the token bridge?

Reply:
You are right! Updated the PR :) Thanks

Reply:
```
worm generate registration --chain acala --module TokenBridge --contract-address 0xe157115ef34c93145Fec2FE53706846853B07F42 --guardian-secret SECRET

010000000001006b565b943c90f3634b8352f9089e3b3a0a50e6f32edea47b91b5a1c15f190a0317a7ffc9fc76df4ed98eb25effe70c0218bdcb16ff9799d1ed8b33e756ca08990100000001000000010001000000000000000000000000000000000000000000000000000000000000000400000000059f9b1700000000000000000000000000000000000000000000546f6b656e427269646765010000000c000000000000000000000000e157115ef34c93145fec2fe53706846853b07f42
```